src/site/layouts/campaign_landing_page.drupal.liquid

```diff
@@ -262,139 +314,159 @@
+                <i aria-hidden="true" class="fab fa-twitter vads-u-padding-right--1"></i>
+                {{ fieldClpConnectWithUs.entity.name }} Twitter
+              </a>
+            </div>
+          </div>
+          <div class="vads-l-col--12 medium-screen:vads-l-col--4">
+            <div class="vads-u-margin-y--1 medium-screen:vads-u-margin-x--1">
+              <a aria-label="{{ fieldClpConnectWithUs.entity.name }} Facebook (opens in a new window)" href="https://facebook.com/{{ socialLinksObject.facebook.value }}" onclick="recordEvent({ 'event': 'nav-hub-rail', 'nav-path': 'Connect with us' });" rel="noreferrer noopener" target="_blank">
+                <i aria-hidden="true" class="fab fa-facebook vads-u-padding-right--1"></i>
+                {{ fieldClpConnectWithUs.entity.name }} Facebook
+              </a>
+            </div>
+          </div>
+          <div class="vads-l-col--12 medium-screen:vads-l-col--4">
+            <div class="vads-u-margin-y--1 medium-screen:vads-u-margin-x--1">
+              <a aria-label="{{ fieldClpConnectWithUs.entity.name }} YouTube (opens in a new window)" href="https://youtube.com/channel/{{ socialLinksObject.youtube.value }}" onclick="recordEvent({ 'event': 'nav-hub-rail', 'nav-path': 'Connect with us' });" rel="noreferrer noopener" target="_blank">
+                <i aria-hidden="true" class="fab fa-youtube vads-u-padding-right--1"></i>
+                {{ fieldClpConnectWithUs.entity.name }} YouTube
+              </a>
+            </div>
+          </div>
+          <div class="vads-l-col--12 medium-screen:vads-l-col--4">
+            <div class="vads-u-margin-y--1 medium-screen:vads-u-margin-x--1">
+              <a aria-label="{{ fieldClpConnectWithUs.entity.name }} Linkedin (opens in a new window)" href="https://linkedin.com/{{ socialLinksObject.linkedin.value }}" onclick="recordEvent({ 'event': 'nav-hub-rail', 'nav-path': 'Connect with us' });" rel="noreferrer noopener" target="_blank">
+                <i aria-hidden="true" class="fab fa-linkedin vads-u-padding-right--1"></i>
+                {{ fieldClpConnectWithUs.entity.name }} Linkedin
+              </a>
+            </div>
+          </div>
+          <div class="vads-l-col--12 medium-screen:vads-l-col--4">
+            <div class="vads-u-margin-y--1 medium-screen:vads-u-margin-x--1">
+              <a aria-label="{{ fieldClpConnectWithUs.entity.name }} Instagram (opens in a new window)" href="https://instagram.com/{{ socialLinksObject.instagram.value }}" onclick="recordEvent({ 'event': 'nav-hub-rail', 'nav-path': 'Connect with us' });" rel="noreferrer noopener" target="_blank">
+                <i aria-hidden="true" class="fab fa-instagram vads-u-padding-right--1"></i>
+                {{ fieldClpConnectWithUs.entity.name }} Instagram
+              </a>
+            </div>
+          </div>
```

Comment:
Previously we had this as a `<ul>` which was great I think but I wanted to use the same grid layout as the other elements and I felt like those grid classes weren't necessarily designed for a list